### PR TITLE
chore: Clean up for latest merged PRs

### PR DESCRIPTION
**Pull request by breathx**

### CODE REVIEW

File: pallets/gear/src/runtime_api.rs [hunk 1]
```diff
@@ -302,6 +302,8 @@ where
             None,
             None,
             payload,
+            // Multiplier 6 was experimentally found as median value between performance,
+            // security and abilities for calculations.
             BlockGasLimitOf::<T>::get() * 6,
```
Comment (ark0f) [inline pallets/gear/src/runtime_api.rs:307]: I guess we can made it as const

**Review by ark0f — APPROVED** [review]

File: pallets/gear/src/runtime_api.rs (hunk 1 above)
Comment (ark0f) [reply pallets/gear/src/runtime_api.rs:307]: Right

File: examples/binaries/read-big-state/src/lib.rs [hunk 2]
```diff
@@ -69,18 +67,13 @@ mod wasm {
     static mut STATE: Option<State> = None;
```
Comment (DennisInSky) [inline examples/binaries/read-big-state/src/lib.rs:67]: Is there any point to have as `Option` at all? Why can't it be initialized statically?

**Review by DennisInSky — APPROVED** [review]

File: examples/binaries/read-big-state/src/lib.rs (hunk 2 above)
Comment (breathx) [reply examples/binaries/read-big-state/src/lib.rs:67]: yeah it may have const constructor but see no difference for test

**Pull request merged**